ui/src/deps/deps.sh

```diff
@@ -10,6 +10,8 @@ if [[ -z ${1} || $envs != *"$1"* ]]; then
   exit 1
 fi
 
+yarn install --frozen-lockfile
```

Comment:
I think this is a better way to solve the issue described [here](https://pmi-engteam.slack.com/archives/C0189PPPKFF/p1694032310063459). Instead of globally installing `openapi-generator-cli` in the Dockerfile (which was breaking stable deployments), run yarn install before installing other dependencies. This installs `openapi-generator-cli` locally instead of globally.

Reply:
This comment thread has been moved [here](https://github.com/all-of-us/workbench/pull/8239#discussion_r1431926648).